resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -613,6 +613,7 @@ class KubernetesSuite extends SparkFunSuite
     Eventually.eventually(TIMEOUT, INTERVAL) {
       assert(kubernetesTestComponents.kubernetesClient
         .pods()
+        .inNamespace(kubernetesTestComponents.namespace)
```

Comment:
This is the same with line 610.
- https://github.com/apache/spark/blob/75291d60d7328dbbf7bd9b130695c4cbcd819d17/resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala#L610